src/main/java/eu/europa/ted/eforms/sdk/analysis/SdkLoader.java

```diff
@@ -183,4 +201,38 @@ public TedefoViewTemplatesIndex getViewTemplatesIndex() throws IOException {
   public Set<TedefoViewTemplateIndex> getViewTemplates() throws IOException {
     return new HashSet<>(getViewTemplatesIndex().getViewTemplates());
   }
+
+  public Set<Codelist> getCodelists()
+      throws IOException, JAXBException, SAXException, ParserConfigurationException {
+    final Set<Codelist> result = new HashSet<>();
+
+    Codelist codelist = null;
+    CodeList codelistXmlPojo = null;
+
+    try (DirectoryStream<Path> dirStream = Files.newDirectoryStream(
+        Path.of(sdkRoot.toString(), SdkResource.CODELISTS.getPath().toString()))) {
+
+      for (Path path : dirStream) {
+        if (!Files.isDirectory(path)) {
+          codelistXmlPojo = XmlParser.loadXmlFile(CodeList.class, path);
+
+          codelist = new Codelist();
+          codelist.setId(path.getFileName().toString().replace(".gc", ""));
```

Comment:
This is not correct: the identifier of a codelist is not its filename, its indicated in the "LongName" element.
See https://docs.ted.europa.eu/eforms/latest/codelists/index.html

Reply:
Fixed